apps/interviewer/src/styles/main.scss

```
@@ -37,7 +37,22 @@ body {
   color: palette('text');
   background: palette('background');
   width: 100%;
+  // `vh` measures the full window on iPadOS 26's resizable windows (which include
```

Stylelint (17.12.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)